astropy/units/equivalencies.py

```diff
@@ -687,3 +687,32 @@ def plate_scale(platescale):
                          "distance/angle")
 
     return [(si.m, si.radian, lambda d: d*platescale_val, lambda rad: rad/platescale_val)]
+
+
+def littleh_as(cosmologyorH0=None):
```

Comment:
Minor: can you put underscores? That is, ``cosmology_or_H0``. I find it quite hard to read otherwise.

Reply:
I still think there should be an underscore between "little" and "h" but that's just my OCD speaking.

Reply:
I can definitely do the first one.  But my argument for `littleh_as` is that apparently unit names don't work with underscores, and the point of the name as it stands now is to directly reference the unit `littleh`. And I think a unit `littleh` and an equivalency operating on it called `little_h_as` is more confusing than the clarity the underscore offers.